### DISCUSSION TITLE
Is there any way to ignore files with flowershow?

### DISCUSSION THREAD

**Discussion by 0xForerunner** [flowershow/flowershow#542] [General and Help]
It would be great to ignore files with a .gitignore style syntax, as well as ignore sections of file with a comment flag, sort of like this:
```
# Interesting Title

Here is some information I want to be exported.

%% BEGIN-EXPORT-IGNORE %%
Here is some information I want hidden.
%% END-EXPORT-IGNORE %%

Here is some other public info.
```

**Comment by rufuspollock** [comment]
Nice suggestion for a feature. Right now we can ignore files (though not via .gitignore syntax) but we can't ignore parts of a file.

**Comment by olayway** [comment]
The OP meant this in the context of Flowershow Self-Hosted which has been deprecated.

For the current Flowershow version (cloud) instructions on excluding files from publishing can be found here https://flowershow.app/blog/how-to-exclude-files-from-publishing.

As for comments (that won't be visible on the published page), you can write them like this (MDX way):
```mdx
{/* comment! */}

{/*
multi
line
comment!
*/}
```

https://flowershow.app/docs/syntax#comments
